samples/bluetooth/mesh/chat/README.rst

```diff
@@ -16,14 +16,19 @@ The Bluetooth® mesh chat sample demonstrates how the mesh network can be used t
 
    chat_cli.rst
 
+FEM support
+***********
+
+.. include:: /includes/sample_fem_support.txt
+
 Requirements
 ************
 
 The sample supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :sample-yaml-rows:
+   :sample-yaml-rows: nrf52dk_nrf52832, nrf52840dk_nrf52840, nrf21540dk_nrf52840
```

Comment:
As far as I know, :sample-yaml-rows: will do the extraction itself from the yaml file, no need to have any additional arguments.

Reply:
One question though... why do we use that in some samples and not in all? A conscious decision, or just something that we weren't thinking of. And if the latter, should we align?

Reply:
Oh, and removing arguments will fix the doc build. Now it's broken :)